src/otx/engine/utils/auto_configurator.py

```diff
@@ -256,7 +257,7 @@ def get_optimizer(self) -> OptimizerCallable:
         Returns:
             OptimizerCallable: The optimizer callable.
```

Comment:
It looks good to add `None` as return Type here. (scheduler as well)

Reply:
Done at https://github.com/openvinotoolkit/training_extensions/pull/2834/commits/dd44ef6ce001cc0715364747f2f7bbab19a0c2af.